controllers/action_controller.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from config import Config
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def create_action_rule(survey_name, period):
+    logger.debug('Finding action plan', survey_name=survey_name, period=period)
+    url = f'{Config.ACTION_SERVICE}/actionplans'
+    response = requests.get(url, auth=Config.BASIC_AUTH)
```

Comment:
Should we also have a try/except here for when it fails to get actionplans?

Reply:
Good point, just raising exception on any error now.